src/index.tsx

```diff
@@ -31,6 +31,31 @@ const isGeoBlocked = async () => {
   }
 }
 
+const isNetBlocked = async () => {
+  if (typeof window === 'undefined') return false
+  const { hostname } = window.location
+  // allow anything when run locally
+  if (hostname === 'localhost' || hostname === '0.0.0.0') return false
```

Comment:
i would just make this a production check - shouldnt be in production code

Reply:
But what about https://dx.dev.gnosisdev.com/ which is `development` but entirely working?

Reply:
ok true